app/models/user.rb

```diff
@@ -93,6 +93,15 @@ def admin_text_search(n)
       (where("name ILIKE ? OR email ILIKE ?", search_str, search_str) +
        joins(:user_emails).where("user_emails.email ILIKE ?", search_str)).uniq
     end
+
+    def ambassadors(organization: nil)
+      ambassador_orgs = Organization.ambassador
```

Comment:
I'm not sure this is possible - but - if it is, I think I'd prefer defining a scope on `Membership` for ambassador organizations, and then using a scope (rather than a class method) that references that scope.

I'm not *completely* sure the scoped solution is much more performant, but I think it looks a lot cleaner. If it's enormously more performant, I'm open to doing the class method.

Reply:
Corollary - I'm not sure I see the value to passing in organization here - because you could just use `organization.users` if you wanted access to those ambassadors

Reply:
Good catch thanks! Updated